### PR TITLE
Programs page

### PR DESCRIPTION
**Pull request by richardverheyen**
This is by no means the last or largest commit, but there's an error with the naming in the programs section which is breaking the images. 
I've changed the names of the programs images to lower case to be in line with the html code, and the problem has been resolved.

### CODE REVIEW

File: Untitled [hunk 1]
```diff
@@ -0,0 +1,426 @@
+<!DOCTYPE html>
```
Comment (janwerkhoven) [inline Untitled]: Hmm don't think we need this file. Could you remove, commit and push?

File: src/templates/about/partners/index.hbs [hunk 2]
```diff
@@ -27,14 +27,16 @@
         <ul>
           {{#partners}}
             <li>
+
               <div class="flip-container">
                 <div class="flipper">
+
```
Comment (janwerkhoven) [inline src/templates/about/partners/index.hbs:33]: Any reason we're adding empty lines?

File: Untitled (hunk 1 above)
Comment (richardverheyen) [reply Untitled]: lol whoops
PageUp strikes again

**Comment by janwerkhoven** [comment]
⛵️

**Pull request merged**